util/argo/argo.go

```diff
@@ -422,8 +422,8 @@ func GetAppProject(app *argoappv1.Application, projLister applicationsv1.AppProj
 		return nil, err
 	}
 	if !proj.IsAppNamespacePermitted(app, ns) {
-		return nil, fmt.Errorf("application '%s' in namespace '%s' is not allowed to use project '%s'",
-			app.Name, app.Namespace, proj.Name)
+		return nil, fmt.Errorf("application '%s' is not allowed to use project '%s'",
```

Comment:
Remove the Application namespace option from the UI 
![Screen Shot 2022-08-29 at 6 34 56 PM](https://user-images.githubusercontent.com/11219262/187370704-c8bac193-f3fd-448e-96c7-0ce9697b7dda.png)